File: Plugins/Aspose_Barcode_Java_for_PHP/src/aspose/barcode/WorkingWithBarcodeImage/BarcodeImageBasicFeatures/GenerateBarcodeWithEmptyBars.php
<?php
namespace Aspose\Barcode\WorkingWithBarcodeImage\BarcodeImageBasicFeatures;

use com\aspose\barcode\BarCodeBuilder as BarCodeBuilder;
use com\aspose\barcode\Symbology as Symbology;

class GenerateBarcodeWithEmptyBars{

    public static function run($dataDir=null){

        # Instantiate barcode object
        $symbology = new Symbology();
        $bb = new BarCodeBuilder("TEXT", $symbology->Code128);

        # Set the FilledBars property to false
        $bb->setFilledBars(false);

        $bb->save($dataDir . "GenerateBarcodeWithEmptyBars.jpg");

        # Display Status.
        print "Generate Barcode With Empty Bars, please check the output file.".PHP_EOL;
    }

}

File: Plugins/Aspose-BarCode-Java-for-Jython/asposebarcode/WorkingWithBarcodeImage/BarcodeImageBasicFeatures/GenerateBarcodeWithEmptyBars.py
from asposebarcode import Settings
from com.aspose.barcode import BarCodeBuilder
from com.aspose.barcode import Symbology

class GenerateBarcodeWithEmptyBars:

    def __init__(self):
        dataDir = Settings.dataDir + 'WorkingWithBarcodeImage/BarcodeImageBasicFeatures/GenerateBarcodeWithEmptyBars'
        
        # Instantiate barcode object
        symbology = Symbology
        bb = BarCodeBuilder("TEXT", symbology.Code128)

        # Set the FilledBars property to false
        bb.setFilledBars(False)

        bb.save(dataDir + "GenerateBarcodeWithEmptyBars.jpg")

        # Display Status.
        print "Generate Barcode With Empty Bars, please check the output file."


if __name__ == '__main__':        
    GenerateBarcodeWithEmptyBars()